packages/bpk-tokens/src/common/base/colors.json

```
@@ -11,16 +11,16 @@
     "BLUE_700": "#008ca8",
     "BLUE_800": "#00758c",
     "BLUE_900": "#005567",
-    "GRAY_50": "#F3F2F5",
-    "GRAY_100": "#E6E4EB",
-    "GRAY_200": "#CCC9D4",
-    "GRAY_300": "#B2AEBD",
-    "GRAY_400": "#9A95A7",
-    "GRAY_500": "#817B8F",
+    "GRAY_50": "#F1F2F8",
+    "GRAY_100": "#DDDDE5",
+    "GRAY_200": "#CDCDD7",
+    "GRAY_300": "#B2B2BF",
+    "GRAY_400": "#8f90a0",
+    "GRAY_500": "#68697F",
     "GRAY_600": "#696179",
```

I'm actually now thinking maybe we should make this the average of 500 and 700 🤔 
```suggestion
     "GRAY_600": "#565770",
```

---

I would suggest we leave it as they are, as this should hopefully mean consumers are forced to change the usage from deprecated values to non deprecated values 😄